src/mono/wasm/Wasm.Build.Tests/Templates/WasmTemplateTestsBase.cs

```diff
@@ -107,14 +107,14 @@ protected ProjectInfo CopyTestAsset(
         bool aot,
         TestAsset asset,
         string idPrefix,
-        bool appendUnicodeToPath = true,
+        bool? appendUnicodeToPath = null,
```

Comment:
`CreateWasmTemplateProject` does the same. We should unify by passing the CI check to `InitProjectLocation` that is the common point for both these methods